pyproject.toml

```
@@ -15,7 +15,7 @@ packages = [
 ]
 [tool.poetry.dependencies]
 python = ">=3.6,<4"
-sphinx = "^4.2.0"
+sphinx = "^5.0.0"
```

```suggestion
sphinx = ">4.2.0"
```
I think this means we are compatible with both 4 and 5 without doing anything.

In order to avoid releasing another version if 6 happens to also be compatible I think it is better to just say >4.2 . what do you think?

---

Thats fine / done: https://github.com/tardyp/sphinx-jinja/pull/36/commits/126577ac011a32912628179e0278c970a69006ee#diff-50c86b7ed8ac2cf95bd48334961bf0530cdc77b5a56f852c5c61b89d735fd711R18